prepare_data/src/test/groovy/org/orbisgis/common/AbstractTablesInitializationTest.groovy

```diff
@@ -24,10 +23,11 @@ class AbstractTablesInitializationTest {
     @Test
     @EnabledIfSystemProperty(named = "test.bdtopo", matches = "true")
     void initParametersAbstract(){
-        H2GIS h2GISDatabase = H2GIS.open("./target/myh2gisbdtopodb")
+        H2GIS h2GISDatabase = H2GIS.open(bdTopoDb-".mv.db", "sa", "")
         def process = PrepareData.AbstractTablesInitialization.initParametersAbstract()
         assertTrue process.execute([datasource: h2GISDatabase])
         process.getResults().each {entry ->
+            println(entry)
```

Comment:
Delete println

Reply:
Yes good catch. It's done in the following commit ;-)